### ISSUE TITLE
LiquidFire tests are breaking against canary at the moment

### ISSUE THREAD

**Issue by acorncom** [ef4/liquid-fire#352]
See this for details:
https://github.com/ember-cli/ember-export-application-global/pull/16

The error we're getting when running either `ember test` or `ember server` is:

    The `initialize` method for Application initializer 'export-application-global' should take only one argument - `App`, an instance of an `Application`.

**Comment by ef4** [comment]
Yeah, we need a fixed ember-export-application-global. Until then, if you take it out of package.json it should unblock development.

**Comment by acorncom** [comment]
I switched to ember `2.0.0` (instead of `canary`) and that resolved it as well.  Just wanted to mention it :-)  Want me to close this issue (or feel free to if you'd like)
